geoips/__version__.py

```diff
@@ -0,0 +1,5 @@
+"""Expose the GeoIPS package version."""
+
+from importlib.metadata import version
+
+__version__ = version(__package__)
```

Comment:
Setting version in __init__.py and trying to import from there elsewhere in GeoIPS caused circular import... Put it in a separate file so it can be imported.